exoscale/validation.go

```diff
@@ -81,3 +81,21 @@ func validateComputeInstanceType(v interface{}, _ cty.Path) diag.Diagnostics {
 
 	return nil
 }
+
+// validateBool validates that the given field is a bool and matches the expected value.
+func validateBool(b bool) schema.SchemaValidateDiagFunc {
+	return validation.ToDiagFunc(func(i interface{}, k string) (s []string, es []error) {
+		value, err := strconv.ParseBool(i.(string))
+		if err != nil {
+			es = append(es, fmt.Errorf("expected type of %s to be bool", k))
```

Comment:
it'd be nice to also wrap `err` in the `Errorf` message

Reply:
done in [9dcf71e](https://github.com/exoscale/terraform-provider-exoscale/pull/409/commits/9dcf71e1adcd1eb0ea33d325924d7822197a9543)